app/models/spree_amazon/response.rb

```diff
@@ -80,6 +80,10 @@ class Authorization < SpreeAmazon::Response
     def response_details
       "AuthorizeResponse/AuthorizeResult/AuthorizationDetails"
     end
+
+    def soft_decline?
+      fetch(response_details, 'SoftDecline').to_s != 'false'
+    end
```

Comment:
is `SoftDecline` truly a string and not a boolean?

Reply:
It's a string